DayrSuryan/DSEthiop11.xml

```diff
@@ -505,6 +513,11 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                    the Evangelist, during the tenure of Coptic Patriarch Cyril V (1874-1927).</origDate>
                    The manuscript was copied in Egypt, by the scribe
                    <persName ref="PRS11958WaldaM" role="scribe">Walda Māryām</persName>.
+
+                  The owner or commissioner of the manuscript might be the person whose name is imprinted on the spine,
+                  cp. <forename xml:lang="ar"> يوسف نجده الحبشي </forename>, that can be interprerted as
+                  the Arabic translation of the name <persName ref="PRS12972Yosef">Yosif ʾAggazaw "the Abyssinian"</persName>.
```

Comment:
here also Yosef, I think. And it sounds maybe a bit too certain to me? Maybe better something "...translation of an Ethiopian name such as...."?

Reply:
I mitigated it a bit.